guide/src/advanced/troubleshooting/clock-drift.md

```diff
@@ -0,0 +1,34 @@
+# Handling Clock Drift
+
+IBC light client security model requires that the timestamp of a header included in client updates for some `client` is within `[now - client.trusting_period, now + client.max_clock_drift)`.
```

Comment:
should ]

Reply:
Hi @DaviRain-Su, I don't believe this is a typo. The `)` on the right-hand side of the range indicates that the max value of the range is exclusive such that it doesn't include `now + client.max_clock_drift`.

Reply:
oh thx your explain, I missunderstand.